### PR TITLE
Land migrated localization for switch page.

### PR DESCRIPTION
**Pull request by Pike**
This is for mozilla/bedrock/issues/8595.

### CODE REVIEW

File: metadata/firefox/switch.json [hunk 1]
```diff
@@ -0,0 +1,72 @@
+INFO 2020-02-27 03:15:09,001 base Raven is not configured (logging is disabled). Please see the documentation for more information.
```
Comment (alexgibson) [inline metadata/firefox/switch.json]: Copy pasta?

**Review by alexgibson — CHANGES_REQUESTED** [review]
Should this also include config files, as per https://github.com/mozmeao/www-l10n/pull/4 ?

File: metadata/firefox/switch.json (hunk 1 above)
Comment (Pike) [reply metadata/firefox/switch.json]: Oops, fixed, sorry.

**Comment by Pike** [comment]
> Should this also include config files, as per #4 ?

Yes, let's do that. I didn't see that they got added to 4, that's why I left them out. I had then locally to test stuff anyway, just didn't `git add` yet ;-)

**Review by alexgibson — APPROVED** [review]

**Pull request merged**